tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist =
+    py{36,37,38}-test{,-devdeps}
+    codestyle
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+isolated_build = true
+
+[testenv]
+changedir = .tmp/{envname}
+description = run tests
+deps =
+    astropy
```

Comment:
@astrofrog , why didn't tox pick up deps from `setup.cfg`? Did I do this wrong?

Reply:
I think we should include:

```
extras = test
```

to pull in the test dependencies from setup.cfg